src/utils/constants.js

```diff
@@ -38,12 +38,12 @@ export const FORM_MESSAGE_TYPES = [
     'success'
 ];
 
-export const ICON_SIZES = [
-    's',
-    'm',
-    'l',
-    'xl'
-];
+export const ICON_SIZES = {
+    's': '12px',
+    'm': '16px',
+    'l': '20px',
+    'xl': '24px'
+};
```

Comment:
Debating removing these like what was done for styles, but for some reason it feels wrong to not have predefined sizes and having the consumer set it.

Reply:
I think icon sizes should not have been removed from fundamental-styles - I think this is fine until they add them back

Reply:
the values should be in `rem` at least:
```
0.75rem
1rem
1.25rem
1.5rem
```